testing/features/automated_release_process.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Automated Release Process
+
+This is to automate the design system removing manual developer actions. 
+    
+As a design system developer 
+I need to be able to specify that a design system release is Alpha or Public
```

Comment:
- specify release number / define as a full or partial release? Not sure what the right terms are here

Reply:
This is a good question. We're using https://github.com/citizensadvice/design-system/releases as the central point for this and that uses the phrase "pre-release". So perhaps "is a stable or pre-release" might do?

Reply:
I didn't use helpful language when we were discussing it - I would go with `pre-release` and `stable release`.

Reply:
Updated, thanks